### PR TITLE
Add issue template and pull request template

### PR DESCRIPTION
**Pull request by wsargent**
Adds an issue and pull request templates, from https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+### Play Version (2.5.x / etc)
```
Comment (marcospereira) [inline .github/ISSUE_TEMPLATE.md]: I would probably start with this question/appeal:

```markdown
### Is this something that you should discuss at our mailing list?

1. [users mailing list](https://groups.google.com/forum/#!forum/play-framework)
2. [developers mailing list](https://groups.google.com/forum/#!forum/play-framework-dev)

If so, you don't need to open this issue.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Pull Request Checklist
+
+* Have you signed the Typesafe CLA? https://www.typesafe.com/contribute/cla
+* Have you squashed your commits?
```
Comment (marcospereira) [inline .github/PULL_REQUEST_TEMPLATE.md]: Relevant link: https://www.playframework.com/documentation/2.4.x/WorkingWithGit#Squashing-commits

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* Have you added copyright headers to new files?
+* Have you checked that both Scala and Java APIs are updated?
+* Have you updated the documentation?
+* Have you added tests?
+
+## Fixes
+
+If this PR fixes an issue, add the URL to the issue here.
```
Comment (marcospereira) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe it is better if the commit message have a "command" to close the issue:

https://help.github.com/articles/closing-issues-via-commit-messages/

Comment (gmethvin) [inline .github/PULL_REQUEST_TEMPLATE.md]: Just ask them to write "Fixes #xxxx" where xxxx is the issue number.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jroper) [inline .github/ISSUE_TEMPLATE.md]: I'm not sure if that quite gives enough context. I'd rephrase it to this:

```
### Are you looking for help?

This is an issue tracker, used to manage and track the development of Play. It is not a support system and so it is not a place to ask questions or get help. If you're not sure if you have found a bug, the best place to start is with either the [users mailing list](https://groups.google.com/forum/#!forum/play-framework) or [Stack Overflow](http://stackoverflow.com/questions/ask?tags=playframework). If you have a feature request, the [developer mailing list](https://groups.google.com/forum/#!forum/play-framework-dev) is a better forum than an issue tracker to discuss it.

Comment (marcospereira) [inline .github/ISSUE_TEMPLATE.md]: :+1:
